core/docker_service.go

```diff
@@ -452,3 +452,18 @@ func (ds *dockerService) getDockerVersionFromCache() (*dockertypes.Version, erro
 	}
 	return dv, nil
 }
+
+func (ds *dockerService) CheckpointContainer(ctx context.Context, options *runtimeapi.CheckpointContainerRequest) (*runtimeapi.CheckpointContainerResponse, error) {
+	return nil, nil
```

Comment:
You need to return a NotImplemented response for all of these, not just nil,nil
```golang
import (
	"google.golang.org/grpc/codes"
	"google.golang.org/grpc/status"
)
```

```suggestion
	return nil, return nil, status.Errorf(codes.Unimplemented, "method CheckpointContainer not implemented")
```

Reply:
fixed